build.sbt

```diff
@@ -60,8 +60,8 @@ libraryDependencies ++= Seq(
   "commons-codec"           % "commons-codec"       % "1.14",
   "com.typesafe.play"       %% "play-mailer"        % "8.0.0",
   "com.typesafe.play"       %% "play-mailer-guice"  % "8.0.0",
-  "com.typesafe.akka"       %% "akka-cluster-tools" % "2.6.1",
-  "com.typesafe.akka"       %% "akka-cluster-typed" % "2.6.1",
+  "com.typesafe.akka"       %% "akka-cluster-tools" % "2.6.17",
```

Comment:
This came downstream I think from the 2.8.8 update.
I guess the play version introduced .17 for 'core akka' (i assume these are plugins). So then needed these plugins to get bumped as well.